app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def links_count
+    current_user.links.count
+  end
+
+  def favourites_links_count
+    current_user.links.where(favourite: true).count
```

Comment:
It should be part of model. Make instance method like `favorite_links` and then call `count` from view